cdk/lib/contributions-referrals.ts

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
+import type { GuStackProps } from "@guardian/cdk/lib/constructs/core";
+import { GuStack } from "@guardian/cdk/lib/constructs/core";
+import type { App } from "aws-cdk-lib";
+import { CfnInclude } from "aws-cdk-lib/cloudformation-include";
+
+export class ContributionsReferrals extends GuStack {
+  constructor(scope: App, id: string, props: GuStackProps) {
+    super(scope, id, props);
+    const yamlTemplateFilePath = join(__dirname, "../..", "cfn.json");
```

Comment:
(Minor) I think either yaml or json is okay here, but as you've migrated to JSON it is worth renaming the variable to indicate that.

Reply:
Done! Thanks @nicl 🙂